src/main/java/seedu/duke/commands/AddFoodCommand.java

```diff
@@ -14,20 +14,22 @@
             + "Trying to add a food item? Use this format:"
             + Ui.LS + MESSAGE_COMMAND_FORMAT;
     public static final String MESSAGE_SUCCESS = "A food item has been added:" + Ui.LS + "%s";
+    public static final String MESSAGE_INVALID_FOOD_CALORIES = "Food calories cannot be less than 0" + Ui.LS
```

Comment:
The message "cannot be less than 0" might be quite misleading as the code doesn't accept 0 too right? 😛